store/sqlstore/upgrade.go

```diff
@@ -173,15 +173,18 @@ func UpgradeDatabaseToVersion33(sqlStore SqlStore) {
 			if err != nil {
 				themeMigrationFailed(err)
 			}
+			defer finalizeTransaction(transaction)
 
 			// increase size of Value column of Preferences table to match the size of the ThemeProps column
 			if sqlStore.DriverName() == model.DATABASE_DRIVER_POSTGRES {
 				if _, err := transaction.Exec("ALTER TABLE Preferences ALTER COLUMN Value TYPE varchar(2000)"); err != nil {
 					themeMigrationFailed(err)
+					return
```

Comment:
`themeMigrationFailed` will actually terminate the process, but I felt it was helpful to make the exit explicit for readers.

Reply:
+1 nit: Maybe a comment, too?